### DISCUSSION TITLE
Can you configure the polling interval on Kargo?

### DISCUSSION THREAD

**Discussion by BenHesketh21** [akuity/kargo#2057] [Q&A] [answered]
I installed v0.6.0 using the helm chart and created a warehouse that looked at 2 images on Dockerhub, I authenticated with Dockerhub to get a higher pull limit but Kargo still got rate limited after 30 minutes-ish. Is this a bug or can I configure Kargo to poll less often, at least for testing purposes?

**Comment by krancour** [comment, marked as answer]
It was added in #2038 and will be in v0.7.0.

**Reply by BenHesketh21** [reply]
Thank you for confirming
